pubspec.yaml

```diff
@@ -5,7 +5,7 @@ homepage: https://github.com/leoafarias/mix
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=2.0.0"
+  flutter: ">=3.3.0"
```

Comment:
We might want to review this constraint as it might impact adoption.

Reply:
This constraint was added for the `strokeAlign` parameter, which hit stable on 3.3.0. It can always be dropped down, but users may encounter issues then